ui/app/templates/components/wizard/policies-others.hbs

```diff
@@ -13,9 +13,6 @@
     </p>
   </WizardSection>
   <WizardSection @headerText="Ready to move on?" @class="wizard-details">
-    <button type="button" class="button next-feature-step" {{action @onAdvance}}>
-      {{@nextFeature}}
-      <Chevron @isButton={{true}} />
-    </button>
+    <Hds::Button @text={{@nextFeature}} @icon="chevron-right" class="next-feature-step" {{on "click" (action @onAdvance)}} />
```

Comment:
it looks like `next-feature-step` relies on `button` 

wonder if we should just do the additional cleanup and remove it? styling will have to be revisited with these wizard components either way 🤷